app/services/activity_logger.py

```diff
@@ -86,7 +86,7 @@ async def wrapper(*args, **kwargs):
             integration_id = str(integration.id) if integration else None
             action_id = func.__name__.replace("action_", "")
             action_config = kwargs.get("action_config")
-            config_data = action_config.data if action_config else {} or {}
+            config_data = action_config.dict() if action_config else {} or {}
```

Comment:
We shouldn't modify anything within services in the integration. I think you may have found a bug, but then let me fix it in the template repo and we sync the fix later after merging this PR.

Reply:
I opened a PR with the fix:
https://github.com/PADAS/gundi-integration-action-runner/pull/10